### PR TITLE
Moves genetics to science entirely.

### PR DESCRIPTION
**Pull request by SergeyMeyer**
# General Documentation
This PR is a work in progress at the moment. I still have to map the rest of the in-rotation maps.
Didn't want to put like two days of work into mapping just to have the PR be closed.
### Intent of your Pull Request
Moves genetics to science.
Wipes genetics' medical access, and takes medical out of their headsets.
Gives paramedics cloning access.
Swaps genetics and the EXPERIMENTOR on Box (Shifts the room a bit so that experimentor isn't just part of medbay)
### Why is this change good for the game?

Genetics is a science job. They literally sit at a computer all shift and press buttons. The most they're expected to do that is medical is help with cloning, but most of the time they don't even do that. From my experience genetics takes the CMO's orders on the science part of their department more than the RD's orders. 
At the moment genetics is placed as a part of medbay on every map. In most maps it literally is just a section of medbay, completely split off from science. Yogs has it split off from science by an airlock.
The work genetics does is science related, not medical related, and the RD should be the one supervising them, not the CMO, who's job is to ensure the crew aren't dying, not that Genetics is researching stuff and not giving hulk to any assistant that asks.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Genetics and Experimentor areas are both moved into different places.
Genetics' supervisors and accesses are changed as well.

### What should players be aware of when it comes to the changes your PR is implementing?
Genetics is now in a different area entirely, while so far experimentors are far more off to the side of science.
### What general grouping does this PR fall under? 
For example, Engineering changes, Medical rebalancing, TEG tweaks, etc.?
Job and mapping changes?
### Are there any aspects of the PR that you would like us not to mention on the Wiki?
Nope
### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
For example, "This new chem will heal X brute damage". 
No numerical values.
# Changelog

Things still needed to be done:

- [ ] Omega
- [x] Delta
- [x] Meta
- [ ] Eclipse

:cl:  
rscadd: Moves genetics into Science completely.
rscadd: Experimentor is moved on Box and Delta so far, probably more maps as well.  
wip: This is work in progress  
/:cl:

### CODE REVIEW

**Comment by N3D6** [comment]
eh, not a huge fan of this. genetics is a hybrid between medical and science, i think its fine that way and doesn't need to change.

**Comment by redmoogle** [comment]
> 
> 
> eh, not a huge fan of this. genetics is a hybrid between medical and science, i think its fine that way and doesn't need to change.

I can agree with you there, but imo I'd find my job easier if I dont have to listen to PLZ FREE MUTATION

**Comment by N3D6** [comment]
> > eh, not a huge fan of this. genetics is a hybrid between medical and science, i think its fine that way and doesn't need to change.
> 
> I can agree with you there, but imo I'd find my job easier if I dont have to listen to PLZ FREE MUTATION

maybe add privacy shutters to genetics or something then, idk. i'm not a fan of removing it from medbay completely because of a few greytiders asking for powers.

**Comment by patpol4** [comment]
I can get behind this. Good change imo, would probably make removing cloning in the future easier

**Comment by SergeyMeyer** [comment]
> > > eh, not a huge fan of this. genetics is a hybrid between medical and science, i think its fine that way and doesn't need to change.
> > 
> > 
> > I can agree with you there, but imo I'd find my job easier if I dont have to listen to PLZ FREE MUTATION
> 
> maybe add privacy shutters to genetics or something then, idk. i'm not a fan of removing it from medbay completely because of a few greytiders asking for powers.

Genetics is a science job. What they do is science. Its listed as a medical and science hybrid because they're supposed to do cloning stuff which they don't even do. And anyways, on every map they're so split off from science that they don't even seem to be part of it.

**Comment by Tipy1802** [comment]
I agree that genetics should be part of science and not part of medbay.
Could you add pictures of the changes?

**Comment by SomeguyManperson** [comment]
Yes move the job that gives people passive benefits for no cost to themsleves into science so they no longer have to deal with pathetic things like "human interaction" and can focus entirely on getting hulk

**Comment by maxion12345** [comment]
Putting the experimentor so close to medbay? Can't really fuck with it now because you risk blowing up and breaching into medbay, a big no-no. As it currently stands, fuckery resulting from experimentor is contained to science (and it's related maint).

**Comment by Jarod1200** [comment]
this is gonna lead to people asking for "science access" so they can powergame the entire department. i think you should leave genetics where it is but give them a reinforced windoor so they can hand out mutators without the clown barging in and taking 5 more mutations when you open the door to hand him the mutator.

**Comment by Tipy1802** [comment]
> this is gonna lead to people asking for "science access" so they can powergame the entire department. i think you should leave genetics where it is but give them a reinforced windoor so they can hand out mutators without the clown barging in and taking 5 more mutations when you open the door to hand him the mutator.

Genetics would still have genetics access so science access won't let you go in the genetics lab. Just getting science access also doesn't let you in any rooms other than RnD and nanites (toxins, xenobio, genetics and all have seperate access leveles).

**Comment by Moja10** [comment]
Y'all newfriends don't even remember why Genetics is 'a part of science' to begin with. It had to do with the old tech tree, where genetic injectors were basically mandatory for getting the good technologies and equipment to print out. Since that was entirely removed when techtree was added and when we rebased, there is actually basically zero reason for genetics to be a part of science lol. It has more to do with medbay as it stands than it does AT ALL with science. 

I also agree with Theos, yes, make them even interact less with other crewmembers and sit in the autism fort that is science. At least they can do SOMETHING useful as it stands when they have access to the cloning room, and can assist with regular medbay needs when shit hits the fan. 

If you don't think genetics relates to medbay well enough, make them relate MORE, not less, by moving them to a department they have literally nothing to do with as it stands. Bad PR

**Comment by Tipy1802** [comment]
Genetics being into science would mean less tiders breaking in or less mutations being spread when they shouldn't be. I am 100% for this. I fucking hate when mutations are widespread. Some fucker made antenna access ALL comms and god is it hell when half of the station has access to secuirty comms. God do I hate un-stunable asistant hulks that break everywhere or people with TK messing with everything (Thanks God for the removal of X-ray, god did I hate X-ray + TK greyshits). There is ONE good mutation in genetics and that's Space Adaptation, everything else is used for harm or shittery in some way,

**Comment by Tipy1802** [comment]
> If you don't think genetics relates to medbay well enough, make them relate MORE, not less, by moving them to a department they have literally nothing to do with as it stands. Bad PR

Genetics relates to medical only through cloning, which in 99% cases is done by the MDs and it seems to be on it's way out. It relates far more to science as it literally does genetics research. Does it connect to the other science sub-departments? Not really but so do xenobio, nanites and robotics. And I can see more ways it can be made to interact with science more than medbay

**Comment by SergeyMeyer** [comment]
> this is gonna lead to people asking for "science access" so they can powergame the entire department. i think you should leave genetics where it is but give them a reinforced windoor so they can hand out mutators without the clown barging in and taking 5 more mutations when you open the door to hand him the mutator.

I've added a reception area to the new genetics area since you've said this. Personally I believe genetics will be forced into more interaction when they're connected to the main hallway instead of being in the back corner of medbay. Anybody who wants genetics items will now have to go and actually talk to them rather than just busting in during the few times they aren't there.

**Comment by ghost** [comment]
if the idea is to reduce greytiders from slamming on the window of the anti social console people, how does putting this in a different department do ANYTHING?

**Comment by JamieD1** [comment]
No, this makes it nanites 2.
